common/gcloud.py

```diff
@@ -131,3 +138,17 @@ def set_default_project(cloud_project: str):
     """Set default project for future gcloud and gsutil commands."""
     return new_process.execute(
         ['gcloud', 'config', 'set', 'project', cloud_project])
+
+
+# pylint: disable=unused-argument
```

Comment:
don't see a point of keeping these unused arguments, since for local we dont need them.
can even rename function to more clear
run_local_instance (since it is just running an instance of startup script locally).

Reply:
OK changed.